app/backend/approaches/chatreadretrieveread.py

```diff
@@ -33,6 +33,7 @@ class ChatReadRetrieveReadApproach:
 Only generate questions and do not generate any text before or after the questions, such as 'Next Questions'"""
 
     query_prompt_template = """Below is a history of the conversation so far, and a new question asked by the user that needs to be answered by searching in a knowledge base about employee healthcare plans and the employee handbook.
+You have access to azure cognitive search index with 100's of documents.
```

Comment:
I assume ChatGPT doesn't care about casing, so better to capitalize for branding reasons?

```suggestion
You have access to Azure Cognitive Search index with 100's of documents.
```

Reply:
Also just curious how you came up with this additional prompt. Was this via testing? Or based on some research?

Reply:
There was one more azure sample that had example of specifying context https://github.com/Azure-Samples/openai/blob/main/Basic_Samples/Functions/functions_with_azure_search.ipynb. Also tested extensively for quality.